pom.xml

```diff
@@ -93,6 +93,11 @@
       <artifactId>javax.ws.rs-api</artifactId>
       <version>2.1.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.json</groupId>
```

Comment:
It seems like we have multiple json libraries now. Can't we just decide on one and limit ourselves to that?
E.g. having a class with just one `String message` field seems like a good solution and is the way many projects do this so they can nicely separate the API specification from the actual code serving it. (i.e. model layer vs controller)

Reply:
Ohh, sure. (I think I did not realise before that the JSONObject library is also implementing JSON Serialization logic.)